src/main/webapp/app/shared/components/code-button/code-button.component.ts

```
@@ -347,4 +384,40 @@
+        const form = document.createElement('form');
+        form.method = 'GET';
+        form.action = reactURL;
+        form.target = newWindow.name;
+
+        // Loop over data element and create input fields
+        for (const key in data) {
+            if (data.hasOwnProperty(key)) {
+                const hiddenField = document.createElement('input');
+                hiddenField.type = 'hidden';
+                hiddenField.name = key;
+                hiddenField.value = Object.getOwnPropertyDescriptor(data, key)!.value;
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)